typescript/infra/config/environments/testnet2/infrastructure.ts

```diff
@@ -0,0 +1,42 @@
+import { InfrastructureConfig } from '../../../src/config';
+
+export const infrastructure: InfrastructureConfig = {
+  kubernetes: {
+    clusterName: 'abacus-testnet',
```

Comment:
`testnet2`?

Reply:
Operationally it's a lot easier to not spin up a new cluster so I'm inclined to just use the same cluster. Setting a cluster up isn't _that_ inconvenient but there are some things like setting up prometheus / configuring grafana charts / setting up external-secrets, etc that are nice to avoid if they're not needed